doc/releases/changelog-dev.md

```diff
@@ -123,45 +57,88 @@
   
   mol = qml.qchem.Molecule(symbols, geometry, basis_name='sto-3g')
   core, one, two = qml.qchem.electron_integrals(mol)()
-  algo = DoubleFactorization(one, two)
   
-  # print the number of non-Clifford gates and logical qubits
-  print(algo.gates, algo.qubits)
+  algo = DoubleFactorization(one, two)
   ```
 
   ```pycon
+  >>> print(algo.gates, algo.qubits)
   103969925, 290
   ```
+  
+  The methods of the [`FirstQuantization`](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.resource.FirstQuantization.html) and the [`DoubleFactorization`](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.resource.DoubleFactorization.html) classes 
+  can be also accessed individually without instantiating an instance of the class:
 
-  The methods of the `FirstQuantization` and the `DoubleFactorization` classes can be also accessed
-  individually. For instance, the logical qubits can be computed by providing the inputs needed for
-  this estimation without initiating the class.
+  - The number of logical qubits with `qubit_cost`
+  - The number of non-Clifford gates with `gate_cost`
 
   ```python
   n = 100000
   eta = 156
   omega = 169.69608
   error = 0.01
-  qml.resource.FirstQuantization.qubit_cost(n, eta, omega, error)
   ```
   
   ```pycon
+  >>> qml.resource.FirstQuantization.qubit_cost(n, eta, omega, error)
   4377
+  >>> qml.resource.FirstQuantization.gate_cost(n, eta, omega, error)
+  3676557345574 
+  ```
+
+<h4>Differentiable error mitigation ⚙️</h4>
+
+* Differentiable zero-noise-extrapolation error mitigation is now available.
+  [(#2757)](https://github.com/PennyLaneAI/pennylane/pull/2757)
+
+  In zero-noise-extrapolation (ZNE), the noise of a circuit is artificially increased by _folding_
+  the circuit. By executing an expectation value with increasing noise, we can extrapolate back to
+  zero noise by means of a polynomial fit. More details on this can be found in the details section
+  of the [documentation](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.transforms.fold_global.html)
+  and a soon-to-appear demo [(PennyLaneAI/qml/529)](https://github.com/PennyLaneAI/qml/pull/529).
```

Comment:
Is the link temporary? Will the demo be merged by the release? Otherwise I would remove the link here.